trickler/scales.py

```diff
@@ -40,6 +40,133 @@ def noop(*args, **kwargs):
     return
 
 
+class GGjj100b:
+    """Class for controlling a G&G JJ100B scale."""
+
+    class ScaleStatusV1(enum.Enum):
+        """Supports the first version of OpenTrickler software."""
+        STABLE = 0
+        UNSTABLE = 1
+        OVERLOAD = 2
+        ERROR = 3
+        MODEL_NUMBER = 4
+        SERIAL_NUMBER = 5
+        ACKNOWLEDGE = 6
+
+    class ScaleStatusV2(enum.Enum):
+        """New version avoids zero (0) which can be confused with null/None."""
+        STABLE = 1
+        UNSTABLE = 2
+        OVERLOAD = 3
+        ERROR = 4
+        MODEL_NUMBER = 5
+        SERIAL_NUMBER = 6
+        ACKNOWLEDGE = 7
+
+    def __init__(self, memcache, port='/dev/ttyUSB0', baudrate=9600, timeout=0.1, _version=1, **kwargs):
+        """Controller."""
+        self._memcache = memcache
+        self._serial = serial.Serial(port=port, baudrate=baudrate, timeout=timeout, **kwargs)
+        # Set default values, which should be overwritten quickly.
+        self.raw = b''
+        self.unit = Units.GRAINS
+        self.resolution = decimal.Decimal(0.02)
```

Comment:
Is this resolution correct for your scale?

Reply:
Negativ, resolution for grains = 0.01